packages/frontend/src/public-pages/components/modals/MenuModal.vue

```diff
@@ -52,8 +52,7 @@
       </router-link>
 
       <router-link
-        v-for="page in pages"
-        v-if="page.enabled"
+        v-for="page in enabledPages"
         :key="page.uuid"
         :to="{ path: page.slug }"
       >
```

Comment:
Should the v-if that is getting removed here be moved down a level so that the functionality remains the same?